app/models/team.rb

```diff
@@ -533,6 +536,9 @@ def filtered_fact_checks(filters = {})
     # Filter by text
     query = query.where('(fact_checks.title ILIKE ? OR fact_checks.url ILIKE ? OR fact_checks.summary ILIKE ?)', *["%#{filters[:text]}%"]*3) if filters[:text].to_s.size > 2
 
+    # Exclude the ones already applied to a target item
+    query = query.where.not('fact_checks.id' => ProjectMedia.find(filters[:target_id].to_i).fact_check_id) unless ProjectMedia.find_by_id(filters[:target_id].to_i).nil?
```

Comment:
if the query run twice in case the item exists then I suggest to do the following:
pm = ProjectMedia.find_by_id(filters[:target_id].to_i)
query = query.where.not('fact_checks.id' => pm.fact_check_id) unless pm.nil?

Reply:
Good idea, done!